src/DwmApi.Desktop/DwmApi.cs

```diff
@@ -77,5 +77,11 @@ public static partial class DwmApi
         public static extern HResult DwmGetColorizationColor(
             out uint pcrColorization,
             [MarshalAs(UnmanagedType.Bool)] out bool pfOpaqueBlend);
+
+        [DllImport(nameof(DwmApi))]
+        public static extern int DwmGetWindowAttribute(IntPtr hwnd, DWMWINDOWATTRIBUTE dwAttribute, out bool pvAttribute, int cbAttribute);
```

Comment:
Also, it seems you've provided two overloads of this method that vary in the type of the `out` parameter. Yet you only provide two overloads although there seems to be other possibly output types as well. To make sure it is sufficiently generalized, we strongly encourage an `out void*` parameter type to match the native definition and leave casting to the caller. We generally discourage many overloads with specific types as well, so these two might collapse to just one, but if these two overloads are by far the most popular, I wouldn't resist leaving them, in addition to the `out void*` one.